src/onboarding-screen/OnboardingScreen.tsx

```diff
@@ -25,6 +24,8 @@ export type OnboardingScreenParams = {
   testID?: string;
 };
 
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
 export const OnboardingScreen = ({
```

Comment:
This is actually a screen component, not a screen?

Reply:
Yes https://github.com/AtB-AS/mittatb-app/pull/4106#discussion_r1440333341